src/hooks/useNextEventWidget.ts

```
@@ -0,0 +1,188 @@
+        }
+
+        return { days, hours, minutes, seconds, isLive: false, isPast: false, relativeText };
+    }, []);
+
+    const fetchNextEvent = useCallback(async () => {
+        setIsLoading(true);
+        setError(null);
+
+        try {
+            const { data: { user } } = await supabase.auth.getUser();
+            if (!user) {
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)